src/pybind/mgr/rook/module.py

```diff
@@ -134,6 +134,20 @@ def wrapper(*args, **kwargs):
     return wrapper
 
 
+class RookEnv(object):
+    def __init__(self):
+        pod_namespace = os.environ.get('POD_NAMESPACE', 'rook-ceph')
+        self.cluster_name = os.environ.get('ROOK_CLUSTER_NAME', pod_namespace)
```

Comment:
This has been renamed to `ROOK_CEPH_CLUSTER_CRD_NAME` in https://github.com/rook/rook/pull/2903

Reply:
Nah, I'm going to wait for another day or two, till I rename them again.

Reply:
Yup, that sounds good, Please wait till it gets merged.